static/src/js/types/sharedTypes.ts

```diff
@@ -41,3 +43,51 @@ export interface UrsProfile {
   /** The email address of the user from the URS profile */
   email_address: EmailAddressString
 }
+
+/** The Collection Metadata */
+export interface CollectionMetadata {
+  [key: string]: unknown
```

Comment:
Make a CMRMetadata type

Reply:
Not going to make a type for this, just change to any with a note about adding types in the future